proof/crefine/ARM_HYP/Tcb_C.thy

```diff
@@ -4509,35 +4509,90 @@ lemma decodeSetTLSBase_ccorres:
   apply (auto simp: unat_eq_0 le_max_word_ucast_id)+
   done
 
+lemma scast_seL4_TCBFlag_MASK_numFlags[simp]:
+  "scast seL4_TCBFlag_MASK = mask numFlags"
+  by (clarsimp simp: seL4_TCBFlag_MASK_def numFlags_def)
```

Comment:
this is probably fine for now, but is skirting close to being an interface lemma for future arch/config links to the C, which might make Wellformed_C a better place for it... but maybe we should leave that until we have more flags